### PR TITLE
furnace: Add version 0.6.5

### LINKED ISSUE #11495: [Request]: Furnace Tracker
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

Furnace 

### Description

a multi-system chiptune tracker compatible with DefleMask modules

### Homepage

https://github.com/tildearrow/furnace

### Download Link(s)

https://github.com/tildearrow/furnace/releases/download/v0.6pre5/furnace-0.6pre5-win64.zip

### Some Indication of Popularity/Repute

1.1k stars on github

### PR DESCRIPTION
**Pull request by moshen**
Add new app: the [furnace](https://github.com/tildearrow/furnace) chiptune tracker

Closes #11495

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/furnace.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        "32bit": {
+            "url": "https://github.com/tildearrow/furnace/releases/download/v0.6.5/furnace-0.6.5-win32.zip",
+            "hash": "f94a951ddb705618fcfe8089244512e5250480c9ea63d7b08fca2098708363ef"
+        }
+    },
+    "bin": "furnace.exe",
+    "shortcuts": [
+        [
+            "furnace.exe",
+            "furnace"
```
Comment (aliesbelik) [inline bucket/furnace.json:20]:
```suggestion
            "Furnace Chiptune Tracker"
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]
Otherwise looks good.

**Comment by moshen** [comment]
Added recommended changes

**Review by aliesbelik — APPROVED** [review]

**Pull request merged**